examples/actors-python/container-env.sh

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright IBM Corporation 2020,2021,2022
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+export KAFKA_BROKERS=kafka:9092
```

Comment:
Why duplicate these into another script?

Reply:
This is used to set the env inside the container used for the python containerized test.

Reply:
ok.  It's probably simpler to have a dedicated script for this, although it would probably also work to set the environment variables in the Dockerfile or in the `docker run` command you have in the makefile.  We change these very rarely, so the duplication isn't that big of a deal.  We just need to remember to also look here (and the ci test should help us remember...).